### PR TITLE
Brigand (Update)

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
- Removed filled salvager rig.
- Removed majority of cargo palettes.
- Removed health analyzer, body bags and saw from medical room.
- Replaced clothing vendomats with random clothing vendomat spawners. Has 5% chance of spawning one of the following machines: ThreadsChoom or BonanzaMatic.
- Replaced mercenary suit storage unit in the armory with random spawner: mercenary hardsuit storage or private security hardsuit storage.
- Added fire alarm.
- Replaced pump and mixer with automatically turned on.
- Increased shuttle price to 61k.

### Bundled changes:
- Added private security hardsuit to the dungeon loot tables.

## Why / Balance
Update

## How to test
1. Buy Brigand from Expedition Shipyard

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl: erhardsteinhauer
- tweak: Minor update to Brigand.

### CODE REVIEW

File: Resources/Prototypes/_NF/Shipyard/Expedition/brigand.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   id: Brigand
   name: NT Brigand
   description: Civilian conversion of an old military light frigate from the early days of humanity's expansion to the stars, predating FTL technology.
-  price: 55010 # ~47835$ on mapinit + ~7175$ from 15% markup
+  price: 53505 # ~46525$ on mapinit + ~6980$ from 15% markup
```
Comment (arimah) [inline Resources/Prototypes/_NF/Shipyard/Expedition/brigand.yml:15]: The only other exped ship in this price range is the Pathfinder, and that's the smallest and cheapest 1–2 person exped ship. It appraises at around 28,845 after purchase, giving it a markup of nearly 83% (!!!). The Brigand gets you _way more stuff_ and, with this proposed pricing change, is _considerably_ better value. You get basically 20k worth of extra stuff for a few hundred spesos more. I think I'd like to see the Brigand price increased _a lot_, to like $60–70k. It's too capable of a vessel to sell for _this little_ compared to other exped ships.

Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Shipyard/Expedition/brigand.yml:15]: I'd say it's Pathfinder issue - no other expedition ship has this big disparity between it's actual price and sell price, not Brigand. In terms of value that Brigand brings to the table: it's barely a step above Pathfinder.

Comment (arimah) [reply Resources/Prototypes/_NF/Shipyard/Expedition/brigand.yml:15]: Also please use a price that's a nice even multiple of 100 or 1000! It's easier to read and those piddly extra spesos make _no difference_.

Comment (ErhardSteinhauer) [reply Resources/Prototypes/_NF/Shipyard/Expedition/brigand.yml:15]: Brigand has it's 15% markup. As any other exped ship, except Pathfinder.

Comment (arimah) [reply Resources/Prototypes/_NF/Shipyard/Expedition/brigand.yml:15]: > it's barely a step above Pathfinder

With the Brigand, you get:
* A mass scanner, camera monitor and loads of exterior cameras
* A ready-made armoury, with restricted access and a hardsuit
* Private quarters for the captain, also with restricted access and a hardsuit
* Dorms that can be used for storage, armoury expansion, whatever else
* An area that can easily be turned into a kitchen
* A medbay with storage, a health analyzer, a body bag, and even a saw (for shotguns, we all know it)
* A salvage techfab
* A telecomms server, prefilled with a common key
* A cell charger right by the airlocks
* A decently sized cargo area with two ATVs and a secret compartment
* FOUR wall rechargers
* Two more hardsuits
* Bilateral blast doors for easier access to planets
* No dirt to clean up or random broken computers everywhere

All for a proposed 585 spesos more. And you don't _lose_ nearly 25k when you sell it unmodified.

Please increase the price to at least 60,000 (~30% markup).

**Review by arimah — CHANGES_REQUESTED** [review]

**Review by arimah — APPROVED** [review]

**Pull request merged**